packages/react/src/components/TimePicker/TimePicker.stories.js

```diff
@@ -89,3 +81,74 @@ export const WithLayer = (args) => {
     </>
   );
 };
+
+export const Playground = (args) => {
+  return (
+    <TimePicker id="time-picker" labelText="Select a time" {...args}>
+      <TimePickerSelect id="time-picker-select-1">
+        <SelectItem value="AM" text="AM" />
+        <SelectItem value="PM" text="PM" />
+      </TimePickerSelect>
+      <TimePickerSelect id="time-picker-select-2">
```

Comment:
```suggestion
      <TimePickerSelect id="time-picker-select-2" disabled={args.disabled}>
```

Can we add this so that the `select` element gets disabled when the playground control is set?